rbc/omnisci_array.py

```diff
@@ -29,6 +30,28 @@ def key(self):
         return self.dtype
 
 
+class Array(object):
+    pass
+
+
+@extending.lower_builtin(Array, types.Any, types.Any)
+def omnisci_array_constructor(context, builder, sig, args):
+    typ = sig.return_type.dtype
+    _, sz = args
+    fa = cgutils.create_struct_proxy(typ)(context, builder) 
+    fa.sz = sz
+    return fa._getpointer()
+
+
+@extending.type_callable(Array)
+def type_omnisci_array(context):
+    def typer(dtype, size):
+        targetinfo = TargetInfo.host()
```

Comment:
Not sure if instantiating the `TargetInfo` here is 100% correct.

Reply:
no, it is not. The targetinfo should contain server info. We'll need to find a way to get the server TargetInfo from RemoteOmnisci instance to here..

Reply:
For instance, via `array_type_converter` function in omnisci_array.py .

Reply:
What is `context` here? If it is RemoteCPUContext instance then `context.target_info` is the solution.

Reply:
Use
```
target_info = context.target_info
```